libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -599,6 +600,9 @@ uint32_t AP_Frsky_Telem::calc_param(void)
     case 5:
         param = (uint32_t)roundf(_battery.pack_capacity_mah(1)); // battery pack capacity in mAh
         break;
+    case 6:
+        param = (uint32_t)ins.get_temperature(0); // IMU temperature in degrees C
```

Comment:
Directly
param = (uint32_t)AP::ins().get_temperature(0);

Reply:
I made a commit to chanche this